public/css/resets.css

```diff
@@ -0,0 +1,48 @@
+* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
You already have an empty reset.css, delete this or move resets.css into the already existing file.